test/models/work_test.rb

```diff
@@ -0,0 +1,26 @@
+require "test_helper"
+
+describe Work do
+  let(:work) {
+    new_work_title = "Star Wars: A New Hope"
+    new_work_creator = "George Lucas"
+    new_work_description = "A young Luke Skywalker is torn from his childhood home at the mysterious appearance of two robots and a wizened sorcerer, one of whom holds the secret to defeating an intergalactic empire."
+    new_work_pub = "1977"
+
+    valid_work_params = {
+      work: {
+        title: new_work_title,
+        creator: new_work_creator,
+        description: new_work_description,
+        publication_year: new_work_pub,
+        category: "movie",
+      },
+    }
+
+    Work.new(valid_work_params)
```

Comment:
The format `valid_work_params` follows the structure for how Rails formats form data when it sends it to the controller via `params` and using our strong params method. Thus; it's not quite the right place to use this form (nested hash) for a model test